src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx

```diff
@@ -111,8 +109,8 @@ describe('LegacyConfigForm', () => {
       // any of the form inputs, this exact object shape is returned back to us, so we're reusing
       // it here.  It's not supposed to be 'the same object', it just happens to be.
       {
-        ...defaultAppConfig,
-        divideByCohorts: true,
+        ...defaultAppConfig(),
+        divideByCohorts: false,
```

Comment:
why was this changed to false?

Reply:
this is changed to false because we are setting a course discussion setting to its initial state. Initially, all toggles would be off. just like `allowAnonymousPosts`.  users can adjust it according to their requirements.